config/routes.rb

```diff
@@ -25,4 +25,6 @@
   get "/shelters/reviews/:review_id/edit", to: 'reviews#edit'
   patch "/shelters/reviews/:review_id", to: 'reviews#update'
   delete "/shelters/reviews/:review_id", to: 'reviews#destroy'
+
+  post "/pets/:pet_id/favorite", to: 'favorites#create'
```

Comment:
Nice! Makes sense to NOT have a 'favorites#new' path like we have had in the past because the favorite is automatically created and no form is generated.